webauthn-rs/src/lib.rs

```diff
@@ -775,7 +836,11 @@ impl Webauthn {
         let extensions = None;
         let credential_algorithms = self.algorithms.clone();
         let require_resident_key = false;
-        let policy = Some(UserVerificationPolicy::Preferred);
+        let policy = if cfg!(feature = "danger-user-presence-only-security-keys") {
+            Some(UserVerificationPolicy::Discouraged_DO_NOT_USE)
```

Comment:
iirc this will hit a `warn!()` in the core library, maybe guard that warning on this feature not being enabled?

Reply:
The warning in core probably isn't needed at this point, so I'll remove it :)